docs/source/transaction_freeze_ledgers.md

```diff
@@ -0,0 +1,36 @@
+# Transaction Freeze Ledgers
+
+#### Why It May Be Needed
```

Comment:
I'm having trouble following what is being said in this section.  I'd like to reword it to be more clear, but I'm afraid I might loose the intent in the process.  Would you be able to take another pass at this clarify things a little more?

Reply:
Agreed. I rewrote this to make it clearer.